### ISSUE TITLE
NTR: dolichol  phosphate mannose biosynthetic process

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28929]
Please provide as much information as you can: 

* **Suggested term label:**

* **Definition (free text)**

* **Reference, in format PMID:#######**

PMID: 18387370

* **Gene product name and ID to be annotated to this term**

DPM1
DPM2
DPM3
DOLK
(human names)

* **Parent term(s)**

?

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

Dol-P-Man synthesis
dolichol  phosphate mannose synthesis

* Can also provide MetaCyc, KEGG, Wikipedia, and other links.

* **Any other information**

major branch point.

background discussion in
https://docs.google.com/presentation/d/1DWPU4TjGO5hmTBKyq_ckGJiT3JIVFf6Oadczb8YrXGQ/edit#slide=id.p

Dolichol phosphate mannose (Dol-P-Man) is a  a precursor to lipid-oligosaccharide intermediates, Protein O-mannosylation, and GPI anchor biosynthesis

**Comment by ValWood** [comment]
Logic definition requires "dolichol phosphate mannose"
https://github.com/ebi-chebi/ChEBI/issues/4553

**Comment by ValWood** [comment]
Thanks!

**Comment by sjm41** [comment]
Looks like Reactome also uses CHEBI:57523:
https://reactome.org/content/query?q=CHEBI%3A57523&species=Homo+sapiens&species=Entries+without+species&cluster=true

In these reactions:
https://reactome.org/content/detail/R-ALL-162681#Homo%20sapiens
https://reactome.org/content/detail/R-ALL-449229#Homo%20sapiens

Which are part of the 'Synthesis of dolichyl-phosphate mannose' pathway:
https://reactome.org/content/detail/R-HSA-162699

_Dolichyl-phosphate-mannose (DPM, DOLPman) is the donor of mannose groups in the synthesis of the dolichyl pyrophosphate-linked precursor oligosaccharide in asparagine-linked glycosylation, in the synthesis of the glycosyl phosphatidylinositol (GPI) anchor precursor, in protein O-mannosylation and in protein C-mannosylation. Its synthesis proceeds in two steps. First, cytosolic GDP-mannose reacts with dolichyl phosphate exposed on the cytosolic face of the endoplasmic reticulum membrane to form DPM with its mannose moiety oriented toward the cytosol. The DPM molecule then flips in the endoplasmic reticulum membrane, so that its mannose moiety is in the endoplasmic reticulum lumen, accessible to the enzymes that catalyze its transfer to growing glycolipids and glycoproteins (Kinoshita and Inoue, 2000; Maeda et al, 2000)._

**Comment by deustp01** [comment]
> Looks like Reactome also uses CHEBI:57523

Yes, we are, because that appears to be the major species of this ionizable compound at pH 7.3 (although there is no comment to that effect in the ChEBI entry. And it doesn't help that RHEa has no entries for either CHEBI:57523 or its unionized form [CHEBI:15809](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI%3A15809) to use as expert guidance.

**Comment by alanbridge** [comment]
Hi, 

just saw this in passing, would https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:58211 be of any use?

 it is dolichyl β-D-mannosyl phosphate, and features in Rhea (https://www.rhea-db.org/rhea?query=CHEBI%3A58211+) and UniProt (https://www.uniprot.org/uniprotkb?query=CHEBI%3A58211).

All the best, Alan

**Comment by deustp01** [comment]
> would https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:58211 be of any use?

I expect it would work for definitions of the new GO term. I need to check our pathway annotation to see if in fact the human pathway / process uses intermediates of specified chain lengths (as annotated) or in reality works for a range of chain lengths as in the suggested term.

**Comment by ValWood** [comment]
OK decision required,
dolichyl D-mannosyl phosphate(1−) CHEBI:57523
dolichyl β-D-mannosyl phosphate [CHEBI:58211](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:58211)

The term is added, I can refine.

@alanbridge thanks for the similarity search ti, very useful!

**Comment by sjm41** [comment]
This question runs into a wider issue we've started to run into elsewhere (eg. #28898)

For BPs, should GO use the specific form of a sugar as used in the RHEA reactions associated with the process/pathway?
If so, then GO should use the more specific dolichyl **β**-D-mannosyl phosphate(1-) CHEBI:58211

Or, for BPs, should GO use a more generic (but still pH7.3) form of the sugar so that the same BP covers a slightly larger range of substrates/products (and/or so that the same BP can be used when authors don't specify or don't know if it's the beta form)? In this case, GO should use the slightly less specific dolichyl D-mannosyl phosphate(1−) CHEBI:57523.

I think the latter may be preferable?
Only problem there is that ChEBI does not appear to have this expected is_a relationship:
```
dolichyl D-mannosyl phosphate(1−) CHEBI:57523
    |_dolichyl β-D-mannosyl phosphate(1-) CHEBI:58211
```
which I suspect is another symptom of the issue described in https://github.com/ebi-chebi/ChEBI/issues/4482

E.g. doing Alan's RHEA search with CHEBI:58211 gives 9 results, but searching with CHEBI:57523 gives 0 results.

**Comment by ValWood** [comment]
ACTION: switch the names and add the synonym

**Comment by ValWood** [comment]
CHEBI
@ValWood The name given is pretty vague and most likely refers to CHEBI:17624 based on evidence found in PMID:28743912. Its major microspecies at pH 7.3 is CHEBI:58211

==

CHEBI:17624 dolichyl β-D-mannosyl phosphate
does have a synonym "dolichol phosphate mannose"
so I will use
CHEBI:58211 dolichyl β-D-mannosyl phosphate(1−)

**Comment by ValWood** [comment]
Hi @pgaudet  I haven't added a biosynthetic process before. Do I need to add
 the dcterms:conformsTo URL the same as the siblings and the equivalence axioms?
I don't see any info here:
https://wiki.geneontology.org/Metabolic_process

**Comment by pgaudet** [comment]
> Do I need to add
the dcterms:conformsTo URL the same as the siblings and the equivalence axioms?

No, this  comes automatically when you add the logical definition.
